src/server/injected/injectedScript.ts

```diff
@@ -198,26 +183,19 @@ export class InjectedScript {
           lastDidNotMatchSelf = element;
         if (matches === 'self' || (matches === 'selfAndChildren' && kind === 'strict'))
           result.push(element);
-        return single && result.length > 0;
       };
 
       if (root.nodeType === Node.ELEMENT_NODE && appendElement(root as Element))
```

Comment:
Since `appendElement` should not ever return anything now, we can change the code:

if (root.nodeType === Node.ELEMENT_NODE)
  appendElement(root as Element);